### DISCUSSION TITLE
load-path and clients

### DISCUSSION THREAD

**Discussion by pedz** [emacs-lsp/lsp-mode#3782] [General]
This started out as a question but I guess I'll add it as a discussion / suggestion.

I'm trying to use [borg](https://github.com/emacscollective/borg) which essentially means I'm trying to configure lsp-mode using [Vanilla Emacs](https://emacs-lsp.github.io/lsp-mode/page/installation/#vanilla-emacs).  Fortunately, I had it working before using [el-get](https://github.com/dimitri/el-get) or otherwise I would be really lost.

el-get's "recipe" adds "." and "clients" to the `load-list` [See here](https://github.com/dimitri/el-get/blob/master/recipes/lsp-mode.rcp#L7).  The Melpa recipe adds [clients/*.el](https://github.com/melpa/melpa/blob/master/recipes/lsp-mode#L3) to `:files`.  It would have helped me if that was mentioned in the docs.  Please consider adding it.

My `lsp-clients` hash is empty because I didn't know to add "clients" to the `load-list`.  Or... perhaps add it to load-paths when lsp-mode is loaded?

**Comment by ericdallo** [comment]
Sure, Just not sure where we should add it, maybe on some building section? https://emacs-lsp.github.io/lsp-mode/page/installation/

**Comment by pedz** [comment]
I submitted a [pull request](https://github.com/emacs-lsp/lsp-mode/pull/3788#issue-1428485471).

**Reply by ericdallo** [reply]
Merged!
